quoting-service/values.yaml

```diff
@@ -106,6 +106,11 @@ config:
   event_trace_state_enabled: true
   event_traceid_per_vendor: false
 
+  endpointSecurity:
+    jwsSign: false
+    fspiopSourceSigningName: 'switch'
+    jwsSigningKey:
```

Comment:
Please add some documentation here, perhaps showing an example of what one would expect the value of `jwsSigningKey` to look like.